tests/e2e/specs/out-of-game/home.spec.js

```diff
@@ -322,52 +322,49 @@ describe('Home - Game List', () => {
         // No open games appear
         cy.contains('[data-cy-join-game]', 'Join Casual').should('not.exist');
         // Existing game is available to spectate
+        cy.wait(500);
```

Comment:
This test was failing because Vuetify's `v-btn-toggle` component  fails to update which option is selected when the button is clicked right after the page loads before it's finished initializing.

We should fix this and remove the need for the wait: #1190